apps/server/src/core/error/loggable/error.loggable.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { NotFound } from '@feathersjs/errors';
+import { BadRequestException, HttpStatus } from '@nestjs/common';
+import { ApiValidationError, BusinessError } from '@shared/common';
+import { ErrorLoggable } from './error.loggable';
+
+class SampleBusinessError extends BusinessError {
+	constructor() {
+		super(
+			{
+				type: 'SAMPLE_ERROR',
+				title: 'Sample Error',
+				defaultMessage: 'sample error message',
+			},
+			HttpStatus.NOT_IMPLEMENTED
+		);
+	}
+}
+
+describe('ErrorLoggable', () => {
+	describe('getLogMessage', () => {
+		describe('when error is an ApiValidationError', () => {
+			const setup = () => {
+				const validationError = { property: 'foo', value: 'bar', constraints: { foo: 'must be baz' } };
+				const error = new ApiValidationError([validationError]);
```

Comment:
Hier wären zwei Error in dem Array eigentlich schöner.

Reply:
Hab ich hinzugefügt.